src/applications/check-in/day-of/pages/Confirmation/TravelPayAlert.jsx

```diff
@@ -4,106 +4,125 @@ import { useTranslation, Trans } from 'react-i18next';
 import ExternalLink from '../../../components/ExternalLink';
 
 const TravelPayAlert = props => {
-  const { travelPayEligible, travelPayClaimError } = props;
+  const {
+    travelPayEligible,
+    travelPayClaimRequested,
+    travelPayClaimError,
+  } = props;
 
   const { t } = useTranslation();
 
-  return (
-    <>
-      {travelPayClaimError ? (
-        <va-alert
-          show-icon
-          data-testid="travel-pay-message"
-          status="warning"
-          uswds
-          slim
+  if (!travelPayClaimRequested) {
+    return (
+      <va-alert show-icon data-testid="travel-pay-info-message" uswds slim>
```

Comment:
From looking at the figma, it seems like this text is no longer wrapped in an alert.

Reply:
fixed